.circleci/config.yml

```diff
@@ -179,14 +189,19 @@ jobs:
           name: generate jazzy documents
           command: |
             currentversion=${CIRCLE_TAG}
-            bash ./Scripts/AWSMobileClientAPIRererence.sh $currentversion
+            bash ./Scripts/AWSMobileClientAPIReference.sh $currentversion
       - run:
           name: copy jazzy documents
           command: |
             git config --local user.name "AWS"
             git checkout  gh-pages
-            mkdir docs/reference/AWSMobileClient
-            cp -R docs/api/* docs/reference/AWSMobileClient
+            if [ -d docs/reference/AWSMobileClient ]    
+            then 
+              echo "docs/reference/AWSMobileClient is present"
+            else
+                mkdir docs/reference/AWSMobileClient
```

Comment:
For future, you don't actually need the if check, assuming that the build systems have a version of `mkdir` that supports the `-p` option: `mkdir -p docs/reference/AWSMobileClient`. The command silently fails without error if the directory already exists, but makes the target with all path elements if not. E.g.:

```bash
% mkdir path/that/doesnt/exist
mkdir: path/that/doesnt: No such file or directory
% echo $?
1
% mkdir -p path/that/doesnt/exist
% echo $?
0
% mkdir -p path/that/doesnt/exist
% echo $?
0
```

Reply:
great! 
I already changed according to your comment